content/en/blog/2020/tradewinds-2020/index.md

```diff
@@ -0,0 +1,196 @@
+enables extension directly in the proxies instead.
+
+Most of Mixer’s use cases for policy enforcement are already addressed with
+Istio's [authentication](/docs/concepts/security/#authentication-policies)
+and [authorization](/docs/concepts/security/#authorization) policies, which
+allow you to control workload-to-workload and end-user-to-workload authorization
+directly in the proxy. Common monitoring use cases have already moved into the
+proxy too - we have
+[introduced in-proxy support](https://istio.io/docs/ops/configuration/telemetry/in-proxy-service-telemetry/)
+for sending telemetry to Prometheus and Stackdriver.
+
+Our benchmarking shows that the new telemetry model reduces our latency
+dramatically and gives us industry-leading performance, with 50% reductions in
+both latency and CPU consumption.
+
+## A new model for Istio extensibility
+
+The model that replaces Mixer uses extensions in Envoy to provide even more
+capability. The Istio community is leading the implementation of a
+[WebAssembly](https://webassembly.org/) (Wasm) runtime in Envoy, which lets us
+implement extensions that are modular, sandboxed, and developed in one of
+[over 20 languages](https://github.com/appcypher/awesome-wasm-langs). Extensions
+can be dynamically loaded and reloaded while the proxy continues serving
+traffic. Wasm extensions will also be able to extend the platform in ways that
+Mixer simply couldn’t.  They can act as custom protocol handlers and transform
+payloads as they pass through Envoy —  in short they can do the same things as
+modules built into Envoy.
+
+We're working with the Envoy community on ways to discover and distribute these
+extensions. We want to make WebAssembly extensions as easy to install and run as
+containers. Many of our partners have written Mixer adapters, and together we
+are getting them ported to Wasm. We are also developing guides and codelabs on
+how to write your own extensions for custom integrations.
+
+By changing the extension model, we were also able to remove dozens of CRDs.
+You no longer need a unique CRD for every piece of software you integrate with
+Istio.
+
+Installing Istio 1.5 with the 'preview' configuration profile won't install
+Mixer. If you upgrade from a previous release, or install the 'default' profile,
+we still keep Mixer around, to be safe. When using Prometheus or Stackdriver for
+metrics, we recommend you try out the new mode and see how much your performance
+improves.
+
+You can keep Mixer installed and enabled if you need it. Eventually Mixer will
+become a separately released add-on to Istio that is part of the
+[istio-ecosystem](https://github.com/istio-ecosystem/).
+
+## Fewer moving parts
+
+We are also simplifying the deployment of the rest of the control plane. To
+that end, we combined several of the control plane components into a single
+binary: `istiod`. This binary includes the features of Pilot, Citadel, Galley,
```

Comment:
I think the current guideline is to use Istiod when talking about the features like Pilot, and _istiod_ when talking about the deployment.

Reply:
That seems like a good guideline. Adjusting.